Makefile

```diff
@@ -24,7 +27,7 @@ test-integration:
 
 .PHONY: install
 install: init
-	cd cmd/ && go install && cd ..
```

Comment:
Every individual command in a target is run in a separate subshell, so you don't need to restore the directory at the end.